routes/Login/createUser.js

```diff
@@ -66,6 +56,28 @@ router.route("/").post(function(req, res) {
 
             res.status(503).send("ERROR: " + err);
         })
+        .then(function() {
+            require('../../imp_services/implogging')(req.cookies.IMPId, function(logService){
+                logService.action.value = username;
+                logService.setType(800);
+                logService.store(function(err, results){
+                    if(err){
+                        res.status(500).send(err);
+                    }
+                    else{
+                        SendConfirmation(email, function (err) {
```

Comment:
does the SendConfirmation() function have to be inside of logService.store(), or can it follow after that in its own .then() promise?